scalikejdbc-core/src/main/scala/scalikejdbc/TxBoundary.scala

```diff
@@ -41,10 +43,23 @@ object TxBoundary {
     }
   }
 
+  private[scalikejdbc] sealed abstract class TxBoundaryMissingImplicits {
+    implicit def ambiguousTxBoundary1[A]: TxBoundary[A] = macro TxBoundaryMissingImplicits.missingTxBoundary[A]
+  }
+  private[scalikejdbc] object TxBoundaryMissingImplicits {
+
+    def missingTxBoundary[A: c.WeakTypeTag](c: Context): c.Expr[TxBoundary[A]] = {
+      import c.universe._
+      c.error(c.enclosingPosition, s"You import TxBoundary.Future._, However any scala.concurrent.ExecutionContext instance could not found in implicit scope.")
```

Comment:
To activate TxBoundary.Future, scala.concurrent.ExecutionContext value is required in implicit scope here.